test/reducers/minions.js

```diff
@@ -14,6 +14,113 @@ describe('minions reducer', () => {
       minionsById: {},
     });
   });
+  it('should handle FREEZE', () => {
+    const summon1 = { minionId: minionId1 } = summon('playerId1', 0, 'CS2_231'); // wisp
+    const summon2 = { minionId: minionId2 } = summon('playerId1', 1, 'EX1_556'); // harvest golem
+    const action1 = freeze(1, minionId1);
+    const action2 = freeze(2, minionId2);
+    const action3 = freeze(3, minionId1, minionId2, 'heroId1', 'heroId2');
+    let minionState = minions(undefined, summon1);
```

Comment:
Don't summon the minions. Use `initialMinionState` from testData.js. If you need more minions to test on you can add them to `initialMinionState`. Just make sure all the other minion tests still pass.